players/computer.py

```diff
@@ -6,11 +6,17 @@ class Computer(Player):
     def __init__(self, marker):
         Player.__init__(self, marker)
         self.rules = GameRules()
+        self.init_db()
 
     def choose_move(self, board):
         opponent = self.switch_marker(self.marker)
-        self.best_move(board, opponent)
-        return self.best_move_var
+        move = self.db.get_best_move_from_db(self.marker, board)
+
+        try:
+            return move[0].best_move
+        except IndexError:
```

Comment:
I like to use exception with truly exceptional cases. Could we handle this without using `try...except`?

Reply:
refactored in https://github.com/olgaboiar/ttt_python/pull/8/commits/5cf705d6fe4c72daa53f230715ab347dc28068e9